defaults/main.yml

```diff
@@ -46,97 +65,89 @@ cis_aide_cron_dow: "*" # Day of week
 cis_aide_cron_dom: "*" # Day of month
 cis_aide_cron_month: "*"
 
-# 1.7.1.2
 cis_local_login_warning_banner: "Authorized uses only. All activity may be monitored and reported.\n"
-# 1.7.1.3
+
 cis_remote_login_warning_banner: "Authorized uses only. All activity may be monitored and reported.\n"
 
-# 2.2.1.1
+fs_mounts:
+  - mountpoint: '/tmp'
+    device: 'tmpfs'
+    fstype: 'tmpfs'
+    opts: 'rw,nosuid,nodev,noexec,relatime'
+  - mountpoint: '/var/tmp'
+    device: 'tmpfs'
+    fstype: 'tmpfs'
+    opts: 'rw,nosuid,nodev,noexec,relatime'
+  - mountpoint: '/dev/shm'
+    device: 'tmpfs'
+    fstype: 'tmpfs'
+    opts: 'rw,nosuid,nodev,noexec,relatime'
+
+# Section 2
 # You should only enable either ntp or chrony, but not both.
 cis_enable_ntp: false # Set to true if ntp should be enabled/configured.
 cis_enable_chrony: true # Set to true if chrony should be enabled/configured.
 
-# 3.4.2
-cis_hosts_allow_all_ips: "127.0.0.1"
+cis_hosts_allowed_ip_range: "127."
 
-# 4.2.3
 # You should only enable either rsyslog or syslog-ng, but not both.
 cis_enable_rsyslog: true # Set to true if rsyslog should be enabled/configured.
 cis_enable_syslog_ng: false # Set to true if syslog_ng should be enabled/configured.
 
-# 4.2.1.4
+# Section 4
 cis_rsyslog_remote_loghost_address: "loghost.example.com" # Can be a hostname or IP address. If no forwarding should occur, exclude this check.
 
-# 4.2.1.5
 cis_rsyslog_accept_remote_messages: false # Set to true if the host should accept remote syslog messages.
 
-# 4.2.2.3
 cis_syslog_ng_file_perms: "0640"
 
-# 5.2.5
+# Section 5
 cis_sshd_max_auth_tries: 4
-
-# 5.2.11
 cis_sshd_ciphers: "aes256-ctr,aes192-ctr,aes128-ctr"
-
-# 5.2.12
 cis_sshd_macs: "hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,umac-128@openssh.com"
-
-# 5.2.13
+cis_sshd_kex_algos: "curve25519-sha256,curve25519-sha256@libssh.org,diffie-hellman-group14-sha256,diffie-hellman-group16-sha512,diffie-hellman-group18-sha512,ecdh-sha2-nistp521,ecdh-sha2-nistp384,ecdh-sha2-nistp256,diffie-hellman-group-exchange-sha256"
```

Comment:
Can we make this `cis_sshd_kex_algorithms`?

Reply:
noted. Will do after the merge.